<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>540</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Scene3D" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <widget class="QMenu" name="menuGraphs">
    <property name="title">
     <string>&amp;Graphs</string>
    </property>
    <addaction name="actionSpeed"/>
    <addaction name="actionAngular"/>
    <addaction name="actionHeight"/>
   </widget>
   <widget class="QMenu" name="menuAnimation">
    <property name="title">
     <string>Анимация</string>
    </property>
    <addaction name="actionBallAnimation"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionLicense"/>
    <addaction name="actionAuthors"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <widget class="QMenu" name="menuLanguage">
    <property name="title">
     <string>&amp;Language</string>
    </property>
    <addaction name="actionEnglish"/>
    <addaction name="actionRussian"/>
   </widget>
   <addaction name="menuGraphs"/>
   <addaction name="menuAnimation"/>
   <addaction name="menuLanguage"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>190</width>
     <height>470</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>9</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Наклонный участок</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <property name="verticalSpacing">
         <number>9</number>
        </property>
        <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Угол наклона [град]</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QSlider" name="horizontalSliderAngle">
          <property name="minimum">
           <number>10</number>
          </property>
          <property name="maximum">
           <number>90</number>
          </property>
          <property name="value">
           <number>69</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="labelAngle">
          <property name="text">
           <string notr="true">69</string>
          </property>
          <property name="minimumSize">
           <size>
            <width>25</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Длина [м]</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QSlider" name="horizontalSliderLength">
          <property name="maximum">
           <number>80</number>
          </property>
          <property name="value">
           <number>20</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="labelLength">
          <property name="text">
           <string notr="true">20</string>
          </property>
          <property name="minimumSize">
           <size>
            <width>25</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Радиус петли [м]</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <property name="verticalSpacing">
         <number>9</number>
        </property>
        <item row="0" column="0">
         <widget class="QSlider" name="horizontalSliderLoopRadius">
          <property name="minimum">
           <number>10</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>20</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="labelLoopRadius">
          <property name="text">
           <string notr="true">2.0</string>
          </property>
          <property name="minimumSize">
           <size>
            <width>25</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Радиус шара [м]</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <property name="verticalSpacing">
         <number>9</number>
        </property>
        <item row="0" column="0">
         <widget class="QSlider" name="horizontalSliderSphereRadius">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>15</number>
          </property>
          <property name="pageStep">
           <number>1</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="labelSphereRadius">
          <property name="text">
           <string notr="true">0.1</string>
          </property>
          <property name="minimumSize">
           <size>
            <width>25</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="title">
        <string>Time [s]</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>9</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item>
         <widget class="QLCDNumber" name="lcdNumber">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>85</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>85</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>25</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>Experiment Speed [%]</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <property name="verticalSpacing">
         <number>9</number>
        </property>
        <item row="0" column="0">
         <widget class="QSlider" name="horizontalSliderSpeed">
          <property name="minimum">
           <number>10</number>
          </property>
          <property name="maximum">
           <number>200</number>
          </property>
          <property name="singleStep">
           <number>1</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
          <property name="sliderPosition">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="invertedControls">
           <bool>false</bool>
          </property>
          <property name="tickPosition">
           <enum>QSlider::NoTicks</enum>
          </property>
          <property name="tickInterval">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="labelSpeed">
          <property name="text">
           <string notr="true">100</string>
          </property>
          <property name="minimumSize">
           <size>
            <width>25</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>8</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QPushButton" name="pushButtonStart">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>27</height>
          </size>
         </property>
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonStop">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>27</height>
          </size>
         </property>
         <property name="text">
          <string>Reset</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="./resources/menuicons/menuicons.qrc">:/menuicons/about.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutRole</enum>
   </property>
  </action>
  <action name="actionAuthors">
   <property name="icon">
    <iconset resource="./resources/menuicons/menuicons.qrc">:/menuicons/people.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Credits</string>
   </property>
  </action>
  <action name="actionSpeed">
   <property name="icon">
    <iconset resource="./resources/menuicons/menuicons.qrc">:/menuicons/speed.png</iconset>
   </property>
   <property name="text">
    <string>Скорость шара</string>
   </property>
  </action>
  <action name="actionAngular">
   <property name="icon">
    <iconset resource="./resources/menuicons/menuicons.qrc">:/menuicons/angular.png</iconset>
   </property>
   <property name="text">
    <string>Угловая скорость шара</string>
   </property>
  </action>
  <action name="actionHeight">
   <property name="icon">
    <iconset resource="./resources/menuicons/menuicons.qrc">:/menuicons/hight.png</iconset>
   </property>
   <property name="text">
    <string>Изменение высоты (y) </string>
   </property>
  </action>
  <action name="actionLicense">
   <property name="icon">
    <iconset resource="./resources/menuicons/menuicons.qrc">:/menuicons/license.png</iconset>
   </property>
   <property name="text">
    <string>&amp;License</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="icon">
    <iconset resource="./resources/menuicons/menuicons.qrc">:/menuicons/qt.png</iconset>
   </property>
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutQtRole</enum>
   </property>
  </action>
  <action name="actionBallAnimation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Вращение шарика</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;English</string>
   </property>
  </action>
  <action name="actionRussian">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Russian</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Scene3D</class>
   <extends>QWidget</extends>
   <header>Scene3D.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>centralwidget</tabstop>
  <tabstop>horizontalSliderAngle</tabstop>
  <tabstop>horizontalSliderLength</tabstop>
  <tabstop>horizontalSliderLoopRadius</tabstop>
  <tabstop>horizontalSliderSphereRadius</tabstop>
  <tabstop>horizontalSliderSpeed</tabstop>
  <tabstop>pushButtonStart</tabstop>
  <tabstop>pushButtonStop</tabstop>
 </tabstops>
 <resources>
  <include location="./resources/menuicons/menuicons.qrc"/>
 </resources>
 <connections/>
</ui>
